### DISCUSSION TITLE
Incremental Backup

### DISCUSSION THREAD

**Discussion by Sunsoft-io** [planetscale/discussion#133] [FAQs] [answered]
Hello,

is there support for incremental backup in PlanetScale?

Thanks...Willi Rauffer, `link-removed`.

**Comment by johnyerhot** [comment, marked as answer]
Hi @Sunsoft-io 👋 

Backups are not incremental. By default everyone will get a daily backup and you can set custom schedules and backup retention on non-free tiers for an additional cost.

On a side note, please do not include links to non-topic or question related content. While you may not mean it as such, we'd like to make sure to keep our discussion area free from anything that may appear to be as advertising. I've edited your question to remove the link.

Thanks!
